File: src/main/java/io/github/oleksivio/telegram/bot/api/model/objects/std/files/inputmedia/InputMedia.java
package io.github.oleksivio.telegram.bot.api.model.objects.std.files.inputmedia;

import io.github.oleksivio.telegram.bot.core.model.ITelegram;

public interface InputMedia extends ITelegram {

    String getType();

    void setType(String type);

    String getMedia();

    void setMedia(String media);

    String getCaption();

    void setCaption(String caption);

    String getParseMode();

    void setParseMode(String parseMode);

}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/objects/std/files/inputmedia/InputMedia.kt
package io.github.oleksivio.telegram.bot.api.model.objects.std.files.inputmedia

import io.github.oleksivio.telegram.bot.core.model.ITelegram

interface InputMedia : ITelegram {

    var type: String?

    var media: String?

    var caption: String?

    var parseMode: String?

}
